### ISSUE TITLE
[enterprise-4.10] Issue in file machine_management/applying-autoscaling.adoc

### ISSUE THREAD

**Issue by mtulio** [openshift/openshift-docs#47273]
### Which section(s) is the issue in?

- [ClusterAutoscaler resource definition](https://docs.openshift.com/container-platform/4.10/machine_management/applying-autoscaling.html#cluster-autoscaler-cr_applying-autoscaling)

### What needs fixing?

The field `utilizationThreshold` is expected to be a string instead of float.

### Steps to reproduce:

1. Create the ClusterAutoscaler manifest (as it is from the doc):

```yaml
apiVersion: "autoscaling.openshift.io/v1"
kind: "ClusterAutoscaler"
metadata:
  name: "default"
spec:
  podPriorityThreshold: -10 
  resourceLimits:
    maxNodesTotal: 24 
    cores:
      min: 8 
      max: 128 
    memory:
      min: 4 
      max: 256 
    gpus:
      - type: nvidia.com/gpu 
        min: 0 
        max: 16 
      - type: amd.com/gpu 
        min: 0 
        max: 4 
  scaleDown: 
    enabled: true 
    delayAfterAdd: 10m 
    delayAfterDelete: 5m 
    delayAfterFailure: 30s 
    unneededTime: 5m 
    utilizationThreshold: 0.4 
```

2. Create the manifest

```bash
$ oc create -f cluster-autoscaler.yaml 
The ClusterAutoscaler "default" is invalid: spec.scaleDown.utilizationThreshold: Invalid value: "number": spec.scaleDown.utilizationThreshold in body must be of type string: "number"
```

### How to fix

1. Change the `utilizationThreshold` to string instead of float

```yaml
(...)
    utilizationThreshold: "0.4"
```

2. Create the ClusterAutoscaler

```bash
$ oc create -f cluster-autoscaler.yaml 
clusterautoscaler.autoscaling.openshift.io/default created

```

**Comment by jeana-redhat** [comment]
Thanks! So rather than just stating a numerical value, you have to put it in quotes?

**Comment by mtulio** [comment]
@jeana-redhat that's correct

**Comment by jeana-redhat** [comment]
Fix merged into `main` and 4.10+ branches
